bundle/libraries/local_path.go

```diff
@@ -77,6 +80,16 @@ func IsLibraryLocal(dep string) bool {
 	return IsLocalPath(dep)
 }
 
+func IsLocalRequirementsFile(dep string) (string, bool) {
+	dep, ok := strings.CutPrefix(dep, "-r")
```

Comment:
should the prefix be /-r\s/ or does "-rrequirements.txt" work?

Reply:
Updated https://github.com/databricks/cli/pull/2736/commits/ee64e50560029a499a95fb4a4646c3a6811bdcae